containers/ecr-viewer/src/app/services/ecrMetadataService.ts

```diff
@@ -44,7 +44,7 @@ interface EcrMetadata {
 }
 
 export interface ERSDInfo {
-  success: boolean;
+  success: boolean | undefined;
```

Comment:
I think this should be boolean, but the field `eRSDInfo?: ERSDInfo` so the `geteRSDInfo` would return `ERSDInfo | undefined`